dotnet/samples/KernelSyntaxExamples/Program.cs

```diff
@@ -18,7 +18,7 @@ public static class Program
     ///     DefaultFilter = "18"    => run only example 18 (also "180" if there is such test)
     ///     DefaultFilter = "chat"  => run all examples with a name that contains "chat"
     /// </summary>
-    public const string? DefaultFilter = "";
+    public const string? DefaultFilter = "Example74";
```

Comment:
Most probably this is accidental change :)
```suggestion
    public const string? DefaultFilter = "";
```

Reply:
This one will be gone after the UT as Tests merge. 😊